From eabbbc1e840dcc403498b78dd009c24d34a4b45b Mon Sep 17 00:00:00 2001
From: Fotios Sioutis <sfotis@gmail.com>
Date: Mon, 14 Nov 2011 22:31:05 +0200
Subject: [PATCH] Macro fixes in Draw package

In draw package there was a case (building with HAVE_NO_DLL outside
cmake build system) that the import or export decorator of a symbol
would not be defined in any case, and result in compiler error.
---
 inc/Draw_Viewer.hxx |    2 ++
 inc/Draw_Window.hxx |    2 ++
 2 files changed, 4 insertions(+), 0 deletions(-)

Index: oce/inc/Draw_Viewer.hxx
===================================================================
--- oce.orig/inc/Draw_Viewer.hxx
+++ oce/inc/Draw_Viewer.hxx
@@ -43,6 +43,8 @@
 # else
 #  define __Draw_API /*__declspec( dllimport )*/
 # endif
+#else
+# define __Draw_API  
 #endif
 #else
 #  define __Draw_API  
Index: oce/inc/Draw_Window.hxx
===================================================================
--- oce.orig/inc/Draw_Window.hxx
+++ oce/inc/Draw_Window.hxx
@@ -204,6 +204,8 @@ void GetNextEvent(Event&);
 # else
 #  define __Draw_API __declspec( dllimport )
 # endif
+#else
+# define __Draw_API
 #endif
 
 // definition de la classe Segment
